src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -377,7 +377,8 @@
   <!-- This target is used to collect the SuggestedWorkload items in the project.-->
   <Target Name="CollectSuggestedWorkloads"
           Returns="@(SuggestedWorkload)"
-          DependsOnTargets="GetSuggestedWorkloads" />
+          DependsOnTargets="GetSuggestedWorkloads"
+          Condition="'$(UsingMicrosoftNETSdk)' == 'true'" />
```

Comment:
In [my repro](https://developercommunity.visualstudio.com/t/the-target-getsuggestedworkloads-does-not-exist-in/1520706?from=email), all the projects are Microsoft.NET.Sdk projects and yet the error remains. I don't see how this added condition will help in my case.

Reply:
Yes, this is incomplete as @davkean and @drewnoakes pointed out. I'm current looking for the proper location for this target.